src/com/rcv/CVRReader.java

```diff
@@ -179,11 +196,10 @@ private CastVoteRecord parseRow(Row castVoteRecordRow, String cvrFileName, int c
           continue;
         } else if (candidate.equals(config.getOvervoteLabel())) {
           candidate = Tabulator.explicitOvervoteLabel;
-        } else if (!config.getCandidateCodeList().contains(candidate)) {
-          if (!candidate.equals(config.getUndeclaredWriteInLabel())) {
-            Logger.warn("no match for candidate: %s", candidate);
-          }
-          candidate = config.getUndeclaredWriteInLabel();
+        } else if (!config.getCandidateCodeList().contains(candidate)
+            && !candidate.equals(config.getUndeclaredWriteInLabel())) {
+          Logger.severe("No match for candidate: %s", candidate);
```

Comment:
In practice, this actually gets really noisy, and is unnecessary in light of the super cool summary error with counts you print out at the end. Recommend removing.